universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java

```diff
@@ -31,49 +30,69 @@ public BaseHtmlLayout(ViewUtils viewUtils) {
     this.viewUtils = checkNotNull(viewUtils);
   }
 
-  /** Returns HTTP content of type "text/html". */
-  public Content htmlContent(DomContent... domContents) {
-    ArrayList<DomContent> contents = new ArrayList<>(Arrays.asList(domContents));
-    ToastMessage privacyBanner =
-        ToastMessage.error(BANNER_TEXT).setId("warning-message").setIgnorable(true).setDuration(0);
-    contents.add(0, privacyBanner.getContainerTag());
-    contents.add(viewUtils.makeLocalJsTag("toast"));
-    contents.add(viewUtils.makeLocalJsTag("radio"));
-    return new HtmlResponseContent(contents.toArray(new DomContent[0]));
+  /** Creates a new {@link HtmlBundle} with default css, scripts, and toast messages. */
+  public HtmlBundle getBundle() {
+    return getBundle(new HtmlBundle());
   }
 
   /**
-   * Returns a script tag that loads Tailwindcss styles and configurations common to all pages in
-   * the CiviForm.
+   * The reason for two getBundle methods here is that order occasionally matters, but this method
```

Comment:
I feel like this docstring should also call out that the subclasses of BaseHtmlLayout should probably almost always, if not just always call this super class version since it adds the tailwind styles and the google analytics, etc.